.nvmrc

```diff
@@ -0,0 +1 @@
+10.22.1
```

Comment:
What's this for?

Reply:
for some reason i couldn't get CRA to run with node v14xx on my local
so i thought just to be safe, we should use the same version i ran create-react-app command with
and this allows you to do `nvm use` and it'll set your node version to the one we need

if you know think we should be able to use a different node version going forward then we could remove

also just found this https://github.com/facebook/create-react-app/issues/9614
might try `npx -p node@LTS -- npx create-react-app my-app --template typescript` next time i'm working on this

Reply:
didn't work 😢